doc/mgmt_preview_quickstart.md

```diff
@@ -0,0 +1,161 @@
+Quickstart Tutorial - Resource Management (Preview Libraries)
+=============================================================
+
+We are excited to announce that a new set of management libraries are
+now in Public Preview. Those packages share a number of new features
+such as Azure Identity support, HTTP pipeline, error-handling.,etc, and
+they also follow the new Azure SDK guidelines which create easy-to-use
+APIs that are idiomatic, compatible, and dependable.
+
+You can find the details of those new libraries
+[here](https://azure.github.io/azure-sdk/releases/latest/#dotnet)
+
+In this basic quickstart guide, we will walk you through how to
+authenticate to Azure using the preview libraries and start interacting
+with Azure resources. There are several possible approaches to
+authentication. This document illustrates the most common scenario
+
+Prerequisites
+-------------
+
+You will need the following values to authenticate to Azure
```

Comment:
This is not strictly required, DefaultAzureCredential would work by just having authenticated VS or VS Code instance or Azure CLI.

Reply:
what if users does not have VS or CLI ?

Reply:
Then we should have a section somewhere for the case, most or our customers do have them and we should show an easiest way to connect.

Reply:
Got it, thanks Pavel

Reply:
I checked this document: https://docs.microsoft.com/en-us/dotnet/api/overview/azure/identity-readme?view=azure-dotnet
and it didn't seem to mention any VS-related setup. It does mention CLI. Considering this was written by a tech writer, I would assume they already knew what the common scenarios are. I think it's best to just leave a link to this pointer: https://docs.microsoft.com/en-us/dotnet/api/overview/azure/identity-readme?view=azure-dotnet